open_lm/datapreprocess/ray/tokenize_shuffle_sbatch.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+#SBATCH -p <partition>
+#SBATCH -t 48:00:00
+#SBATCH --job-name=ray_tokenize
+#SBATCH --nodes <N>
+#SBATCH --ntasks-per-node 1
+#SBATCH --cpus-per-task=128
+#SBATCH --output=%x_%j.out
+
+export PATH="~/miniconda3/condabin:$PATH"
+source ~/miniconda3/etc/profile.d/conda.sh
+conda activate open_lm  # install according to tng/tools/environment.yml
+cd $OPEN_LM_BASE
+
+# The following is the recommended setup from the Ray documentation.
+
+# Getting the node names
+nodes=$(scontrol show hostnames "$SLURM_JOB_NODELIST")
+nodes_array=($nodes)
+head_node=${nodes_array[0]}
+head_node_ip=$(srun --nodes=1 --ntasks=1 -w "$head_node" hostname --ip-address)
+
```

Comment:
suggested change:
allowing to set the temp dir enables to specify which storage on the cluster should be used; /tmp/ is usually local to the node, if this is too small, then a global /scratch/.. directory can be used
export TEMP_DIR=/tmp/ray